lldb/test/API/tools/lldb-server/commandline/TestGdbRemoteConnection.py

```diff
@@ -75,7 +75,7 @@ def __init__(self):
     class Pipe(object):
         def __init__(self, prefix):
             while True:
-                self.name = "lldb-" + str(random.randrange(1e10))
+                self.name = "lldb-" + str(random.randrange(int(1e10)))
```

Comment:
How about `random.randrange(10000000000)`?

Reply:
I would prefer to leave it as the original author wrote it if that's alright. I'm assuming they found the shorthand more readable

Reply:
FWIW I find it easier to read `e10` than `0000000000`, so I'd also keep the style as is.